### PR TITLE
Docs.docker compose

### PR DESCRIPTION
**Pull request by enguerran**
Updates documentation about *Docker Compose* trying to reflect https://gist.github.com/enguerran/0eb3943ccb6dd6fe8cec5663a628f177.

- [X] Add documentation.
- [X] Add your name in the contributors file.

r? @Natim

### CODE REVIEW

File: docs/tutorials/install.rst [hunk 1]
```diff
@@ -114,6 +114,44 @@ with a *PostgreSQL* container.
...
+       - "8888:8888"
+      volumes:
+        - ./config:/etc/kinto
+
+Note that ``config/kinto.ini`` must define the following options
+(where ``postgres:postgres`` is the user/password you defined in ``docker-compose.yml``):
+
+::
+
+    kinto.cache_backend = kinto.core.cache.postgresql
```
Comment (Natim) [inline docs/tutorials/install.rst]: I think you might want to define this using environment variables don't you?

Comment (enguerran) [inline docs/tutorials/install.rst]: In fact, that is exactly why I created the gist.
I needed to customize the `kinto.ini` file and I saw that the `kinto.ini` generated in the docker container was referencing to memory storage.
I do not want to set kinto using some options via environment variables and some via a `kinto.ini` file.
So, I changed the `docker-compose.yml` to exclude the environment variables and set up a *docker volume* to mount the kinto configuration file into the container.

Comment (Natim) [inline docs/tutorials/install.rst]: Oh but you configure the PostgreSQL server with Docker don't you?

File: docs/tutorials/install.rst [hunk 2]
```diff
@@ -114,6 +114,44 @@ with a *PostgreSQL* container.
     wget https://raw.githubusercontent.com/Kinto/kinto/master/docker-compose.yml
     sudo docker-compose up
 
+If you want to change the settings, you need to mount a custom settings file
+into the *Kinto* container. Hopefully Docker Compose let you do that the exact
```
Comment (leplatrem) [inline docs/tutorials/install.rst]: typo: lets you (?)

File: docs/tutorials/install.rst [hunk 3]
```diff
@@ -114,6 +114,44 @@ with a *PostgreSQL* container.
...
+same way Docker does (assuming you have created the config file ``./config/kinto.ini``):
+
+::
```
Comment (leplatrem) [inline docs/tutorials/install.rst]:
```
.. code-block:: yaml 
    :emphasize-lines: 12,13
```
?

File: docs/tutorials/install.rst [hunk 4]
```diff
@@ -114,6 +114,44 @@ with a *PostgreSQL* container.
...
+    web:
+      image: kinto/kinto-server
+      links:
+       - db
+      ports:
+       - "8888:8888"
+      volumes:
+        - ./config:/etc/kinto
+
+Note that ``config/kinto.ini`` must define the following options
```
Comment (leplatrem) [inline docs/tutorials/install.rst]: Note that with the above example, ``config/kinto.ini`` must define...

File: docs/tutorials/install.rst [hunk 5]
```diff
@@ -114,6 +114,44 @@ with a *PostgreSQL* container.
...
+    kinto.storage_backend = kinto.core.storage.postgresql
+    kinto.storage_url = postgres://postgres:postgres@db/postgres
+    kinto.permission_backend = kinto.core.permission.postgresql
+    kinto.permission_url = postgres://postgres:postgres@db/postgres
+
+Now you can:
+
+- Stop the containers with ``docker-compose stop``.
+- Start the containers with ``docker-compose up -d`` (``-d`` is for background/daemon).
+- Connect to postgres with ``docker-compose exec --user postgres db psql``.
```
Comment (leplatrem) [inline docs/tutorials/install.rst]: nit: PostgreSQL service

File: docs/tutorials/install.rst [hunk 6]
```diff
@@ -114,6 +114,44 @@ with a *PostgreSQL* container.
...
+::
+
+    kinto.cache_backend = kinto.core.cache.postgresql
+    kinto.cache_url = postgres://postgres:postgres@db/postgres
+    kinto.storage_backend = kinto.core.storage.postgresql
+    kinto.storage_url = postgres://postgres:postgres@db/postgres
+    kinto.permission_backend = kinto.core.permission.postgresql
+    kinto.permission_url = postgres://postgres:postgres@db/postgres
+
+Now you can:
```
Comment (leplatrem) [inline docs/tutorials/install.rst]: This is not completely related to settings customization and also applies when the yml file is download with wget. What do you think if we move it above?

File: docs/tutorials/install.rst [hunk 7]
```diff
@@ -114,6 +114,44 @@ with a *PostgreSQL* container.
     wget https://raw.githubusercontent.com/Kinto/kinto/master/docker-compose.yml
     sudo docker-compose up
 
+If you want to change the settings, you need to mount a custom settings file
+into the *Kinto* container. Hopefully Docker Compose let you do that the exact
+same way Docker does (assuming you have created the config file ``./config/kinto.ini``):
```
Comment (leplatrem) [inline docs/tutorials/install.rst]: . The above downloaded `docker-compose.yml` file can be adjusted this way:

File: docs/tutorials/install.rst (hunk 3 above)
Comment (enguerran) [inline docs/tutorials/install.rst]: I do not know rST very well, so I *cargo cult*-ed the block code syntax. Is it wrong?

File: docs/tutorials/install.rst (hunk 1 above)
Comment (enguerran) [inline docs/tutorials/install.rst]: The PostgreSQL server is a container started by docker-compose.yml with a configuration through environment variables:

```yml
db:
    image: postgres
    environment:
        POSTGRES_USER: myusername
        POSTGRES_PASSWORD: mysecuredpassword
```

File: docs/tutorials/install.rst (hunk 7 above)
Comment (enguerran) [inline docs/tutorials/install.rst]: You mean, we should change the `docker-compose.yml` file in the repository to match this one:

```yml
    db:
      image: postgres
      environment:
        POSTGRES_USER: postgres
        POSTGRES_PASSWORD: postgres
    web:
      image: kinto/kinto-server
      links:
       - db
      ports:
       - "8888:8888"
      volumes:
        - ./config:/etc/kinto
```

File: docs/tutorials/install.rst (hunk 6 above)
Comment (enguerran) [inline docs/tutorials/install.rst]: Yeah, that is true, it is some common command lines to control our running containers.

Comment (enguerran) [inline docs/tutorials/install.rst]: I added another usefule command line: https://github.com/Kinto/kinto/pull/936/commits/9cc476182e0085cb18d15d5713d977e7b2ecd4b5#r89494120

File: docs/tutorials/install.rst (hunk 5 above)
Comment (enguerran) [inline docs/tutorials/install.rst]: fixed wih the new commit

File: docs/tutorials/install.rst (hunk 2 above)
Comment (enguerran) [inline docs/tutorials/install.rst]: fixed wih the new commit

**Comment by leplatrem** [comment]
Thanks!

**Pull request merged**